compose-profiles/test-local-3nodes/docker-compose.tools.yml

```diff
@@ -16,9 +16,9 @@ services:
       file: ../docker-compose-definitions.yml
       service: medco-loader
     environment:
-      - DB_NAME=i2b2medcosrv0
+      - I2B2_DB_NAME=i2b2medcosrv0
```

Comment:
Here should also be overriden the `GA_DB_NAME` variables for all 3 medco-loader services, to the `gamedcosrvX` values.